### PR TITLE
Bump Compute to beta version

### PR DESCRIPTION
**Pull request by andreamlin**
Following pattern from https://github.com/googleapis/google-cloud-java/pull/4581

### CODE REVIEW

**Comment by andreamlin** [comment]
@kolea2  PTAL

File: google-cloud-clients/google-cloud-compute/pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-clients</artifactId>
-    <version>0.83.1-alpha-SNAPSHOT</version><!-- {x-version-update:google-cloud-clients:current} -->
+    <version>0.83.1-beta-SNAPSHOT</version><!-- {x-version-update:google-cloud-clients:current} -->
```
Comment (sduskis) [inline google-cloud-clients/google-cloud-compute/pom.xml]: Please revert this line, since `google-cloud-clients` is still at alpha.

**Comment by sduskis** [comment]
@andreamlin, there are some additional changes required before this will build.

**Comment by sduskis** [comment]
@andreamlin, this PR is quite stale at this point.  Would it be ok to close this?

**Comment by andreamlin** [comment]
Sure.

**Pull request closed without merging**